explorer/explorertypes.go

```diff
@@ -67,15 +67,15 @@ type TxInfo struct {
 }
 
 type TicketInfo struct {
-	TicketMaturity               int64
-	TimeTillMaturity             float64 //TimeTillMaturity represents the time before a particular ticket reaches maturity
-	PoolStatus                   string
-	SpendStatus                  string
-	TicketPoolSize               int64   //TicketPoolSize represents the total number of block before the average win
-	TicketExpiry                 int64   //TicketExpiry represents the total number of blocks before a ticket expires
-	ConfirmsToPoolSizePercentage float64 //ConfirmsToPoolSizePercentage represents the percentile average of the confirmations and TicketPoolSize
-	TicketExpiryDaysLeft         float64 //TicketExpiryDaysLeft represents the approximate days left before the given ticket expires
-	AveWinTime                   float64 //ClosureToAverageWinDaysLeft represent approximately how close the ticket is to the average win in days
+	TicketMaturity          int64
+	TimeTillMaturity        float64 //TimeTillMaturity represents the time before a particular ticket reaches maturity
+	PoolStatus              string
+	SpendStatus             string
+	TicketPoolSize          int64   //TicketPoolSize represents the total number of ticket in the pool
```

Comment:
well, this is the target ticket pool size, right?  We're not actually putting the current ticket pool size in this variable.